services/wallet/token/token.go

```diff
@@ -174,8 +215,12 @@ func NewTokenManager(
 	accountsDB *accounts.Database,
 	tokenBalancesStorage TokenBalancesStorage,
 ) *Manager {
+	err := loadTokensStoreDetails()
```

Comment:
accessing the filesystem in the constructor is a no-go for me, better leave all these heavy operations for `Start()`

Reply:
@dlipicar I know that, the only reason why I consciously add it here is to have `TokensSources` ready before creating stores in case somewhere we use those details but they are not ready yet.

Reply:
Also as mentioned in [this](https://github.com/status-im/status-go/pull/6363#discussion_r1972413393) comment all that will be changed soon, cause we will have auto-refresh tokens and for that PR we will keep fetched token details in the db. 

We started with this like building the auto-refresh feature, but then gave up on that and decided just to add aave list. But today concluded to have the auto-refresh feature.